app/javascript/src/clone.ts

```diff
@@ -1,3 +1,3 @@
-export function clone<T>(obj: T): T {
+export function clone(obj: any): any {
```

Comment:
I can't think of any case where something is fixed by switching to any (just covered up).  Is it a compile time error?